deploy/charts/kiali-operator/templates/istio-destinationrule.yaml

```diff
@@ -0,0 +1,16 @@
+{{ if eq .Values.istio.ingress.enabled true }}
```

Comment:
Regarding the destination rule/gateway/virtual service, we can leave this here since it is the "istio way" of exposing the Kiali server, however, the operator does provide a k8s-native way (using an Ingress) and an OpenShift-native way (using Route) - that's what this setting does:

https://github.com/kiali/kiali-operator/blob/v1.18.0/deploy/kiali/kiali_cr.yaml#L230-L233

Perhaps in the future the operator could be told to create an ingress using the "istio" way, but that doesn't need to hold up this PR.

Reply:
Completely agree, it's quite new for Istio to recommend it to be done this way. In the older versions Kubernetes ingress resources were also just created. 

I can open up an issue to add in that feature perhaps?

Reply:
> I can open up an issue to add in that feature perhaps?

Feel free to add a future enhancement request issue.